docs/tutorial/population_dynamics.rst

```diff
@@ -0,0 +1,75 @@
+.. warning::
+    This is early access alpha functionality. While usable, this has not yet been tested in a large number of different
+    scenarios. The API is still be subject to change *without any prior deprecation notice*! If you use this
+    functionality keep a close eye on the changelog for any changes that may affect your analysis.
+
+Population Dynamics
+===================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+
+The following tools enable analysis of experiments which can be described as a system of discrete states. Consider as an example
+a DNA hairpin which can exist in a folded and unfolded state. At equilibrium, the relative populations of the two states are governed by the interconversion kinetics described by the rate constants \\(k_{fold}\\) and \\(k_{unfold}\\)
+
```

Comment:
I think for here, I'd like to keep the physics background discussion to a minimum and focus on the code features. This may be more applicable when a specific kinetics section is added or a specific example case for instance.